src/components/Pill/Pill.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styles from './Pill.scss';
+
+type PillProps = {
+  /**
+   * String or node to be rendered inside the pill.
+   */
+  children: React.ReactNode;
+  /**
+   * Icon to display before the children.
+   */
+  icon?: string;
+  pillStyle?: 'default' | 'alt';
```

Comment:
Can we please call these "faint" and "inverted", and make `pillStyle` required?

There's only one usage of `<Pill>` in the wild. The name "default" implies we should respect that one usage ... and I don't think it's worthy of any reverence :).

Reply:
will do. looks like it's used in `LivingBriefing`. Do you know any examples of this I can look at to test the change?

<img width="639" alt="Screen Shot 2022-03-23 at 9 44 08 AM" src="https://user-images.githubusercontent.com/139535/159751753-c7b168aa-dd34-415b-b05a-d0d996897e8f.png">

Reply:
Found it!

Reply:
updated!